### PR TITLE
Workaround to install OKD opearators

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: roles/ci_local_storage/tasks/cleanup.yml [hunk 1]
```diff
@@ -44,29 +44,41 @@
...
+      {{
+          hostvars | dict2items |
+          selectattr("value.ansible_hostname", "defined") |
+          selectattr("value.ansible_hostname", "in", k8s_nodes_hostnames) |
+          map(attribute="key") |
+          list
+      }}
 
-- name: Delete directories on worker nodes
+- name: Create directories on worker node
```
Comment (lewisdenny) [inline roles/ci_local_storage/tasks/cleanup.yml:75]: Small nit, this should be `Delete directories on worker node`

Comment (pablintino) [reply roles/ci_local_storage/tasks/cleanup.yml:75]: Good one.

File: scenarios/reproducers/networking-env-definition.yml
Comment (cjeanner) [inline scenarios/reproducers/networking-env-definition.yml:1]: you want to remove that one at some point I think ;).

Comment (pablintino) [reply scenarios/reproducers/networking-env-definition.yml:1]: I won't commit both commits all-together, so I'll remove it at that point.

**Comment by cescgina** [comment]
I was looking a the failures and the kuttl job runs the `crc_storage` target (and the cleanup one) internally https://github.com/openstack-k8s-operators/ci-framework/blob/370b0ff4d62e99e5794a6438950c7f5249a0a127/ci/playbooks/kuttl/run-kuttl-tests.yml#L16-L28, so it's bound to run into problems. It would probably be ok to merge this breaking it and then moving those calls to the role in a followup

**Comment by cjeanner** [comment]
@cescgina good catch yes!

Hopefully #1344 should solve the issue.

**Comment by pablintino** [comment]
Not needed anymore.

**Pull request closed without merging**